src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -203,6 +220,17 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
             return response;
         }
 
+        private static bool GetQueryParameterValue(HttpRequestMessage request, string queryParameterName)
```

Comment:
Since this method is only concerned with boolean query string parameters, I would rename it to `GetBooleanQueryParameterValue`.

Reply:
@cgillum  Great point! I will change it.